days-since-start-of-the-year.js

```diff
@@ -0,0 +1,12 @@
+const now = new Date();
+const currentTimestamp = now.getTime();
+const daysSinceTheStartOfTheYear = () => {
+  const then = new Date("2021-01-01");
+  const landingTimestamp = then.getTime();
+  const millisecondsInADay = 1000 * 60 * 60 * 24;
+  const days = Math.floor(
+    (currentTimestamp - landingTimestamp) / millisecondsInADay
+  );
+  return `${days}`;
```

Comment:
Here you are converting a number to a string which imho is unnecessary. You could just return the number instead:

```suggestion
  return days;
```